src/components/shared/CheckboxField/index.test.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import CheckboxField from './index';
+
+describe('The Checkbox Field component', () => {
+  it('renders without crashing', () => {
```

Comment:
I'll add some more test.

The "renders without crashing" is a nice smoke test. I didn't check the `value` because most of the props map 1:1 to the `<input />`. It doesn't hurt to add additional tests, but I don't think it adds a ton of value.